charts/devlake/values.yaml

```diff
@@ -332,7 +331,7 @@ ingress:
   enabled: false
   enableHttps: false
   # Set to false if you want to use a different ingress controller
-  useDefaultNginx: true
+  useDefaultNginx: false
```

Comment:
If this is no longer used in template, you may want to remove it?
```suggestion
```

Reply:
it would be used when using an external grafana and the ingress class is nginx